.chloggen/fix_prometheus_recceiver_honor_labels.yaml

```
@@ -0,0 +1,29 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: prometheusreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Group scraped metrics into resources created from `job/instance` label pairs
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [34237]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: The receiver will now create a resource for each distinct job/instance label combination. | 
+  In addition to the label/instance pairs detected from the scraped metrics, a resource representing the overall |
+  scrape configuration will be created. This additional resource will contain the scrape metrics, such as the number of scraped metrics, the scrape duration, etc.
```

```suggestion
subtext: |
  The receiver will now create a resource for each distinct job/instance label combination.
  In addition to the label/instance pairs detected from the scraped metrics, a resource representing the overall
  scrape configuration will be created. This additional resource will contain the scrape metrics, such as the number of scraped metrics, the scrape duration, etc.
```
I'm not 100% sure how the current formatting will work, but I'm worried it won't render correctly. This is the typical YAML multiline syntax I've seen.